.github/workflows/daily.yml

```diff
@@ -0,0 +1,34 @@
+name: Daily - Do npm install and run all unit tests
+
+on:
+  schedule:
+    # run daily at midnight
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: [14]
```

Comment:
Should we include node 12 as well ?

Reply:
By default Runtime uses Node v14 and we are making sure that our npm doesn't break on it for a daily run.
Not sure we need to add more versions to matrix for daily run as long as build and test of PRs and commits includes all supported versions.